go/lambda/README.md

```diff
@@ -0,0 +1,99 @@
+# AWS SDK for Go code examples for AWS Lambda
+
+## Purpose
+
+These examples demonstrate how to perform several Lambda operations.
+
+## Prerequisites
+
+You must have an AWS account, and have your default credentials and AWS Region
+configured as described in
+[Configuring the AWS SDK for Go](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html)
+in the _AWS SDK for Go Developer Guide_.
+
+## Running the code
+
+### ConfigureFunction/ConfigureFunction.go
+
+This example enables an Amazon S3 bucket to send notifications to a Lambda function.
+
+`go run ConfigureFunction.go -f FUNCTION -a ARN`
+
+- _FUNCTION_ is the name of the Lambda function.
+- _ARN_ is the ARN of the S3 bucket sending a notification to the function.
+
+The unit test mocks the service client and the `AddPermission` function.
+
+### MakeFunction/MakeFunction.go
+
+This example creates a Lambda function.
+
+`go run MakeFunction.go -z ZIPFILE -b BUCKET -f FUNCTION -h HANDLER -a ROLE-ARN -r RUNTIME`
+
+- _ZIPFILE_ is the name of the zip file, without the .zip extension, containing the Lambda function code.
+- _BUCKET_ is the name of an Amazon S3 bucket in the same region as the Lambda function.
+- _FUNCTION_ is the name of the Lambda function.
+- _HANDLER_ is the name of the method within your code that Lambda calls.
+- _ROLE-ARN_ is the ARN of the function's execution role.
```

Comment:
...the **Amazon Resource Name (ARN)**...

Reply:
Fixed